jupyterhub/handlers/pages.py

```diff
@@ -678,4 +678,5 @@ def get(self):
     (r'/token', TokenPageHandler),
     (r'/error/(\d+)', ProxyErrorHandler),
     (r'/health$', HealthCheckHandler),
+    (r'/api/health$', HealthCheckHandler),
```

Comment:
Adding the `/api/` prefix to health and metrics is required for api-only operation. These should probably have been on `/api/` all along, but it never really had a consequence until now.